### ISSUE TITLE
Graph section with SignalPlot

### ISSUE THREAD

**Issue by Intonjokel** [swharden/ScottPlot#596]
Is it possible to display only the most current section of the graph with PlotSignal ? That I could pan to the left if necessary. In the example code everything is piled up and squeezed on one window.

Thx

**Comment by swharden** [comment]
Hi @Intonjokel,

From your question it sounds like you are trying to plot live incoming data, but you only want the latest 500 points to be displayed. You can achieve this by setting the signal plot's `minRenderIndex` and `maxRenderIndex` to only span the index range you want to display. To automatically adjust the axis limits to fit new data you can call `plt.AxisAuto()`.

Let me know if this doesn't solve your issue 👍

**Comment by Intonjokel** [comment]
Yes, exactly. I would like to display live data. I have already tried minRenderIndex and maxRenderIndex. Unfortunately, only this part of the data is displayed and I have no possibility to see the whole values from 0 on. Is it possible that I only see the partial section and if necessary I can see the values up to time 0?

**Comment by swharden** [comment]
Hi @Intonjokel,

Setting `minRenderIndex` and `maxRenderIndex` will cause a signal to only display data points between those index values. It sounds like you actually want all data points to be displayed.

When you say you only want to "see" the last 500 points, perhaps this can be achieved by setting the axis limits to only surrounded the latest data? This way all points are rendered (and can be accessed with the mouse click/drag), but the "view" is centered on the last 500 points.

```cs
// inspect the values of your data to determine which limits to use
formsPlot1.plt.Axis([xMin, xMax, yMin, yMax]);
formsPlot1.Render();
```

Does this solve your problem?

**Comment by Intonjokel** [comment]
Yeaaaahh!!!! Thanks a lot. That was my problem.

Thank u very much!
